tests/modules/blockonkeys.c

```diff
@@ -350,7 +350,7 @@ int bpoppush_reply_callback(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int
 int bpoppush_timeout_callback(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
     VALKEYMODULE_NOT_USED(argv);
     VALKEYMODULE_NOT_USED(argc);
-    return ValkeyModule_ReplyWithSimpleString(ctx, "Request timedout");
+    return ValkeyModule_ReplyWithSimpleString(ctx, "Request timed out");
```

Comment:
This is a user error message, so we should avoid changing it in case anyone has taken a character-by-character dependency on it.